spec/build-spec.js

```diff
@@ -35,6 +37,8 @@ describe('Build', () => {
         return specHelpers.vouch(fs.realpath, dir);
       }).then( (dir) => {
         directory = dir + '/';
+        const createdHomeDir = temp.mkdirSync('atom-build-spec-home');
+        os.homedir = () => createdHomeDir;
```

Comment:
Yes, I was going to do it like that but then I saw all the others tests doing sync, I will change that